repos/system_upgrade/el7toel8/actors/bindupdate/libraries/updates.py

```diff
@@ -0,0 +1,86 @@
+from leapp.libraries.common import isccfg
+from leapp.libraries.stdlib import run
+
+# Callback for walk function
+callbacks = {
+    'dnssec-lookaside': isccfg.ModifyState.callback_comment_out,
+}
+
+
+def paths_from_issues(issues):
+    """ Extract paths from list of BindConfigIssuesModel """
+    paths = []
+    for issue in issues:
+        paths.append(issue.path)
+    return paths
+
+
+def parser_file(parser, path):
+    for cfg in parser.FILES_TO_CHECK:
+        if cfg.path == path:
+            return cfg
+    return None
+
+
+def debug_log(log, text):
+    if log is not None:
+        log.debug(text)
+
+
+def make_backup(path, backup_suffix='.leapp'):
+    """ Make backup of a file before modification """
+    backup_path = path + backup_suffix
+    run(['cp', '--preserve=all', path, backup_path])
+
+
+def update_section(parser, section):
+    """ Modify one section
+    :ptype section: ConfigSection
```

Comment:
`ptype` - I've never seen this before. Also googling didn't help. Maye you meant just `:type section: ...`

Reply:
Okay, I thought I read that somewhere. But type is probably correct version.